Lib/random.py

```diff
@@ -238,19 +238,19 @@ def __init_subclass__(cls, /, **kwargs):
                 break
 
     def _randbelow_with_getrandbits(self, n):
-        "Return a random int in the range [0,n).  Returns 0 if n==0."
+        "Return a random int in the range [0,n).  Returns 0 if n<=1."
```

Comment:
This change is unnecessary, and confusing. `n == 0` is a genuine special case that it makes sense to call out in the docstring. `n == 1` is not a special case, and the behaviour for `n == 1` is already covered by the "Return a random int in the range [0, n)" part of the docstring.

Reply:
Ah.. OK, I understand.